ci/scripts/build-linux.sh

```diff
@@ -27,7 +27,7 @@ function install_platform_specific_dependencies() {
   # Currently lantern_extras binaries are only available for Linux x86_64
   # We won't install onnxruntime as lantern_extras are used only for external index in tests
   pushd /tmp
-    LANTERN_EXTRAS_VERSION=0.0.6
+    LANTERN_EXTRAS_VERSION=0.0.12
```

Comment:
Are there any downsides to pulling the latest version of extras via github API ([https://api.github.com/repos/lanterndata/lantern_extras/releases/latest](https://api.github.com/repos/lanterndata/lantern_extras/releases/latest)) and using that in the build here?


[1]: https://api.github.com/repos/lanterndata/lantern_extras/releases/latest